docs/design/eth-subscribe.md

```diff
@@ -9,12 +9,12 @@ The JSON-RPC Relay currently doesn't support subscription using websocket as is
 1. Implement websocket subscription in the relay
 2. Introduce two new methods - `eth_subscribe` and `eth_unsubscribe`
 3. Support subscription for event logs
+4. Support subscription for latest blocks `newHeads`
 
 ## Non-Goals
 
 1. Add long-term memory to the relay
 2. Reduce the efficiency of the relay or mirror node due to high resource usage
```

Comment:
Maybe not related but "reduce the efficiency"?

Reply:
This was written as an Non-Goal for the original `eth_subscribe` for logs implementation.  I added the requirement to monitor calls to `eth_getBlockByNumber` to detect latency in the `newHeads` feature.